sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestBySqlAggregator.java

```diff
@@ -168,12 +168,12 @@ private static class EarliestByLatestBySqlAggFunction extends SqlAggFunction
           InferTypes.RETURN_TYPE,
           OperandTypes.or(
               OperandTypes.sequence(
-                  "'" + aggregatorType.name() + "(expr, timeColumn)'\n",
+                  "'" + StringUtils.format("%s_BY", aggregatorType.name()) + "(expr, timeColumn)'\n",
```

Comment:
these have single quotes surrounding them while others do not, is there a _right_ way to do this? I know its basically entirely cosmetic on what shows up in validation exceptions, but just wondering if there is any sort of standard as the javadocs don't seem to call out what the right thing to do is.

Also how do things that are not using `OperandTypes.sequence` present themselves?

Reply:
@clintropolis, yes, the single vs double quotes inconsistency is everywhere in this area. I'm not sure there's a "right" way to do this since there seems to be exceptions inside the Calcite layer itself (`GROUPING`, for example). For the most part when `OperandTypes.sequence` is not present, it comes back with single quotes, but again there are exceptions😅 . I guess one option is to sanitize the strings by trimming the single quotes (or include them), but I don't feel too strongly about it. We could also do it in the web-console if needed. What do you think?

Reply:
also, fwiw, I'm intending to use this function signature by calling `org.apache.calcite.sql.type.SqlOperandTypeChecker#getAllowedSignaturesDruidOperatorTable` outlined in the [`INFORMATION_SCHEMA.ROUTINES` proposal]( https://github.com/apache/druid/issues/14346). So that's another use case in addition to how we surface validation exceptions.

Reply:
>For the most part when OperandTypes.sequence is not present, Calcite gives back a string with single quotes in them, but again there are exceptions for things defined in Calcite internally

if calcite built-ins use single quotes then we should probably add single quotes in our customized ones to be consistent-ish

Reply:
I scraped and fixed the ones where there is an inconsistent usage. Specifically, the inconsistent usages include:
- No or incorrect single quotes around function names
- No space between args
- Additional `\n`